lib/ApplicationFeatures/GreetingsFeature.cpp

```diff
@@ -36,16 +36,20 @@ void GreetingsFeature::prepare() {
   LOG_TOPIC("e52b0", INFO, arangodb::Logger::FIXME)
       << "" << rest::Version::getVerboseVersionString();
 
+  // building in maintainer mode or enabling catch test code will incur runtime overhead,
+  // so warn users about this
+  bool warn = false;
 #ifdef ARANGODB_ENABLE_MAINTAINER_MODE
-  LOG_TOPIC("0458b", WARN, arangodb::Logger::FIXME)
-    << "==========================================================";
```

Comment:
why no stripes ?

Reply:
Wanted to convert the multiple log messages into just a single one, to reduce the log spam.
There are people out there that trigger alerts on each individual log message of type "WARN". 
For those it is better to issue just a single log message and not 4.

Reply:
but this is a non-production message

Reply:
btw, we're still missing a way not to output several log messages for a multiline javascript log message - i.e. very bad for the wintendos, since it creates one windows eventlog entry for each line!

Reply:
@graetzer: Ok, the banner is back
@dothebart: I won't address any JavaScript logging in this PR.

Reply:
javascript already uses the bulk interface, the logging facility splits it into multiple messages.